src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx

```diff
@@ -0,0 +1,148 @@
+    )
+
+    return artworksForSeoProduct.map(({ node }, index) => {
+      const { artists, is_price_range, partner, price } = node
+      const location = partner.locations && partner.locations[0]
+      const artistsName = artists
+        ? toSentence(node.artists.map(({ name }) => name))
+        : null
+      const isInstitution = partner.type === "Institution"
+
+      return (
+        <Product
+          key={node.__id}
+          data={{
+            name: node.title,
+            image: node.image.url,
+            description: node.meta.description,
+            url: `${APP_URL}${node.href}`,
+            brand: {
+              "@type": "Person",
+              name: artistsName,
+            },
+            ...(isInstitution
+              ? {}
+              : {
+                  category: node.category,
+                  productionDate: node.date,
+                  offers: {
+                    "@type": "Offer",
+                    price: !is_price_range
+                      ? formatCurrency(price)
+                      : {
+                          minPrice: formatCurrency(price.split("-")[0]),
+                          maxPrice: formatCurrency(price.split("-")[1]),
+                        },
+                    priceCurrency: node.price_currency,
+                    availability: AVAILABILITY[node.availability],
+                    seller: {
+                      "@type": "ArtGallery",
+                      name: partner.name,
+                      image: partner.profile.icon.url,
+                      address: location
+                        ? [
+                            location.address,
+                            location.address_2,
+                            location.city,
+                            location.state,
+                            location.country,
+                            location.postal_code,
+                          ]
+                            .filter(Boolean)
+                            .join(", ")
+                        : null,
+                      telephone: location ? location.phone : null,
+                    },
+                  },
+                }),
+          }}
+        />
+      )
+    })
+  }
+}
+
+export const SeoProductsForArtworks = createFragmentContainer(SeoProducts, {
+  artworks: graphql`
+    fragment SeoProductsForArtworks_artworks on FilterArtworks
+      @argumentDefinitions(
+        first: { type: "Int", defaultValue: 30 }
+        after: { type: "String", defaultValue: "" }
+      ) {
+      artworks_connection(first: $first, after: $after) {
+        edges {
+          node {
+            __id
+            availability
+            category
+            date
+            href
+            is_acquireable
+            is_price_range
+            price
+            price_currency
+            title
+            artists {
+              name
+            }
+            image {
+              url(version: "large")
```

Comment:
We may not have to use the `large` image. let me know if this needs to be updated.

Reply:
this can probably use the `thumbnail` size, I believe that is what we use elsewhere in JSONld